Wiki/de/index.md

```diff
@@ -0,0 +1,215 @@
+viele verschiedene Arten von Projekten unterstützen zu können. Die Zahl derer 
+wird im Grunde nur durch die momentan aktiven BOINC Projekte und durch die 
+"project whitelist", über welche die Community abstimmt, begrenzt.
+
+Es gab in der Vergangenheit einen Wechsel vom originalen [Gridcoin-Classic](gridcoin-classic "wikilink")
+zu "Gridcoin-Research", der eigentlichen Gridcoin. Gridcoin-Classic nutzte einen 
+Proof-of-Work Hybriden, wodurch die Computer die halbe Rechenzeit mit "Hashing" beschäftigte,
+während sie die andere Hälfte ihren Beitrag zur Wissenschaft leisteten; Zwar eine
+Verbesserung zu Bitcoin, es wurden aber immer noch Computerressourcen vergeudet.
+Die BOINC Arbeit wurde nur dazu genutzt, um die Beihilfe zu messen, welche mit dem
+Block geschürft wurde. 
+
+Gridcoin-Research macht verschlüsseltes Mining und Proof-of-Work hashing obsolet.
+Mit dem Wechsel zu Gridcoin-Research und damit zum Proof-of-Stake Algorithmus, wird
+nun nahezu die ganze Energie der wissenschaft übergeben. Die Rechenarbeit die jetzt
+zur Auffrechterhaltung der Blockchain notwendig ist, ist minimal.
+
+
+
+## Metrics
+
+**[Gridcoin-Classic](gridcoin-classic "wikilink") - Retired**
+
+  - Start: 16. Oktober 2013
+  - Ende: 20. April 2015
+  - Von: Rob Halförd
+  - Algorithmus: Scrypt Proof-of-Work
+  - Angebot: 40 Million Approx.
+  - Block Zeit: 2.5 Minutes
+  - Retarget: Every 30 Minutes
+  - Belohnung: variabel
+  - Vorgefertigt: 0%
+
+**Gridcoin-Research Proof-of-Stake**
+
+  - Start: 11. Oktober 2014
+  - Von: Rob Halförd
+  - Algorithmus: Proof of Stake. Belohnung basierend auf 10 GRC + die geleistete
+    Forschungsarbeit auf BOINC
+  - Angebot: 10 GRC je neuem Block (~9600 GRC pro Tag) und ~28000 neue GRC für
+    Forschungsbelohnungen
+  - Block Zeit: 90 Seconds
+  - Vorgefertigt: 0% - Die anfängliche Verteilung basierte auf Proof-of-Burn von
+    Gridcoin Classic - Research Foundation Block = 340,569,880
+
+## P2P
+
+Gridcoin ermöglicht peer to peer (P2P) Transaktionen weltweit und nahezu in
+Echtzeit, wofür nur sehr geringe Gebühren fällig werden.
+Der Code ist open source und erlaubt es jedem am Erfolg und in der 
+Entwicklung teilzuhaben. Open Source ist in der Regel sicherer, da
+jeder den Code einsehen kann und ihn im Bedarf verbessern kann.
+
+Der Erfolg von BOINC beruht hauptsächlich auf seinem Credit System, welches
+einen Wettbewerb zwischen den Nutzern ermöglicht. Gridcoin erweitert dies,
+indem es einen austauschbaren Maßstab hinzufügt. Teilnehmer können nun
+ihren eigenen Beitrag einsehen und haben mit der Blockchain ein Dokument für
+den wissenschaftlichen Fortschritt.
+
+
+## Vergleich zu Bitcoin
+
+Bitcoin verbraucht geschätzt 59.31 TWh, laut
+[Digiconomist](https://digiconomist.net/bitcoin-energy-consumption).
+Diese Zahl wird zwar viel diskutiert, sie zeigt aber sicherlich, dass
+eine Menge Energie von Bitcoin's SHA-256 Algorithmus verbraucht wird.
+Diese Energie könnte für weitaus sinnvollere Kalkulationen verwendet werden.
```

Comment:
```suggestion
eine Menge Energie von Bitcoins SHA-256 Algorithmus verbraucht wird.
Diese Energie könnte für weitaus sinnvollere Berechnungen verwendet werden.
```

Reply:
I would suggest to use "Rechenleistung" instead of "Energie"

Reply:
Changed =)